Index: e_config.c
===================================================================
--- e_config.c	(revision 56746)
+++ e_config.c	(working copy)
@@ -928,13 +928,17 @@
    E_Config *tcfg = NULL;
 
    e_config = e_config_domain_load("e", _e_config_edd);
+   if (!e_config) 
+     printf("DEBUG: No e_config from domain load. This is BAD\n");
    if (e_config)
      {
         int reload = 0;
 
+        printf("DEBUG: Have e_config. Checking Version\n");
         /* major version change - that means wipe and restart */
 	if ((e_config->config_version >> 16) < E_CONFIG_FILE_EPOCH)
 	  {
+             printf("DEBUG: e_config too old. Updating\n");
 	     /* your config is too old - need new defaults */
 	     _e_config_free(e_config);
              e_config = NULL;
@@ -952,6 +956,7 @@
         /* config is too new? odd! suspect corruption? */
 	else if (e_config->config_version > E_CONFIG_FILE_VERSION)
 	  {
+             printf("DEBUG: e_config too new. WTF\n");
 	     /* your config is too new - what the fuck??? */
 	     _e_config_free(e_config);
              e_config = NULL;
@@ -969,6 +974,7 @@
           {
              /* your config is so old - we don't even bother supporting an
               * upgrade path - brand new config for you! */
+             printf("DEBUG: e_config way too old. You need a new one\n");
 	     _e_config_free(e_config);
              e_config = NULL;
              reload = 1;
@@ -984,6 +990,7 @@
           }
         if (reload)
           {
+             printf("DEBUG: Doing e_config reload\n");
              e_config_profile_del(e_config_profile_get());
              e_config = e_config_domain_load("e", _e_config_edd);
           }
