diff --git a/e/src/bin/e_module.c b/e/src/bin/e_module.c
index 85787f7..2e2c197 100644
--- a/e/src/bin/e_module.c
+++ b/e/src/bin/e_module.c
@@ -629,6 +629,7 @@ _ignore_cb(void *data, E_Dialog *dialog)
 static void
 _e_module_whitelist_check(void)
 {
+   return; // DO NOT LAUNCH 3RD PARTY MODULE POPUP. WORKAROUND FOR BODHI LINUX
    Eina_List *l, *badl = NULL;
    E_Module *m;
    unsigned int known = 0;
